tsweb/varz/varz.go

```diff
@@ -10,7 +10,6 @@ import (
 	"fmt"
 	"io"
 	"net/http"
-	_ "net/http/pprof"
```

Comment:
`pprof` was imported in varz used, but not completely clear why, I am opting to remove it because we are not allowed to use it in the client path when the target is js/wasm.

Alternatively, the metrics can be included via build tags.

To verify that it was not use, check the `/debug/metrics` endpoint with and without this package, and did not see any noticeable difference and all the same metrics were reported.